src/node/markdown/plugins/highlightLines.ts

```diff
@@ -4,7 +4,7 @@
 
 import MarkdownIt from 'markdown-it'
 
-const RE = /{([\d,-]+)}/
+const RE = /{([\w,-]+)}/
 
```

Comment:
@brc-dd Only thing I am concerned about here is, I am allowing any word or character in it. I have tried this pattern `/{([(\d|\bshowLineNumbers\b),-]+)}/` but it didn't work. Please, let me know about a better regex pattern

Reply:
I ended up using this - `/{(([\d,-]|\bshowLineNumbers\b)+)}/`